client/scala/armada-scala-client/build.sbt

```diff
@@ -20,7 +20,7 @@ Compile / PB.targets := Seq(
 Compile / PB.protoSources ++= Seq(file("./proto"))
 
 libraryDependencies ++= Seq(
-    "io.grpc" % "grpc-netty" % scalapb.compiler.Version.grpcJavaVersion,
+    "io.grpc" % "grpc-netty" % "1.71.0",
```

Comment:
As I mentioned in Slack:
The version picked here should work best with Armada and for any user, it should not be specifically picked for Spark. If Spark does not work with it, we need to deal with it where Spark comes into the game (Spark cluster manager), not here. This project should be Spark-agnostic.